candidates/templates/candidates/constituencies.html

```diff
@@ -18,13 +18,13 @@
 for that post:{% endblocktrans %}</p>
 
 <ul>
-  {% for post_extra in all_constituencies %}
+  {% for post in all_constituencies %}
```

Comment:
Fixed.

I was planning to squash all the view updates into a single commit before merging so the commit message issue should go away.